applications/nrf_desktop/description.rst

```diff
@@ -952,6 +952,24 @@ See :ref:`app_build_file_suffixes` and :ref:`cmake_options` for information abou
 
 .. nrf_desktop_fastpair_important_end
 
+nRF54L MCUboot provisioning
+===========================
+
+nRF54L-based nRF Desktop devices enable hardware cryptography for the MCUboot bootloader.
+The public key that MCUboot uses to validate the application image is securely stored in the hardware Key Management Unit (KMU).
+In this use case, the application image is automatically signed by the |NCS| build system.
+However, the public key is not automatically provisioned to the device when programming the bootloader and the application images using the ``west flash`` command.
+
+To provision the MCUboot keys, use the ``west ncs-provision`` command before programming the bootloader and application images.
```

Comment:
Wouldn't it make more sense to add this documentation section to general MCUboot documentation and only provide a link here?

Reply:
It would be a bit more complicated to make a generic documentation for MCUboot (using KMU is controlled with a Kconfig option, multiple public keys might be used etc.). I would prefer to keep this simple docs in scope of nRF Desktop for NCS 2.9. After MCUboot bootloader docs are improved, we could remove part of this documentation and replace it with a link.